amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
```

Comment:
re: schemas, namespace encapsulation is a nice thing. That is unfortunate dbschema does not properly support functionality which spans 75% of its name.